.github/workflows/deploy-scripts-internally.yaml

```diff
@@ -34,7 +34,7 @@ jobs:
 
                     cd /tmp/nango-temp/nango-integrations
                     npm install zod soap botbuilder
-                    nango admin:deploy-internal template-$integration
+                    nango admin:deploy-internal template-$integration || true
```

Comment:
This will ignore all legitimate errors :/ 
Also why not all integrations would have an env? That seems to be what we want + solve this issue

Reply:
The integration might not exist in the shared account meaning there aren't credentials in the shared dev account. For example we don't have algolia credentials in the shared dev account. Because of that the entire script fails and won't continue past that. 

For the context of this action it is OK to ignore this error and other legitimate errors as the only goal is to deploy the scripts that exist in the shared dev account.

Reply:
can we `have deploy-internal` return a different error code when integration doesn't exist and check the error code?

Reply:
added logic to check the error code and return true if `unknown_provider_config`